ui/desktop/app/components/settings-card/server/index.hbs

```diff
@@ -0,0 +1,41 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+<SettingsCard
+  @header={{t 'settings.server.title'}}
+  @icon='cloud'
+  @description={{t 'settings.server.description'}}
+>
+  <:state>
+    {{#if @model.serverInformation}}
+      <Hds::Badge
+        @text={{t 'settings.connected'}}
+        @icon='check'
+        @color='success'
+      />
+    {{/if}}
+  </:state>
+  <:body>
+    <div>
+      <Hds::Text::Display>
+        {{t 'settings.provider'}}
+      </Hds::Text::Display>
+      <Hds::Text::Body @tag='p' @weight='medium' data-test-server-name>
+        {{this.providerName}}
+      </Hds::Text::Body>
+    </div>
+    <div class='large-column'>
+      <Hds::Text::Display>
+        {{t 'settings.url'}}
```

Comment:
non-blocking: to be consistent with the "Welcome to Boundary" card should we use "Cluster URL" here as well?

Reply:
good point, checking with design

Reply:
done, thank you!